intranet/apps/bus/views.py

```diff
@@ -2,15 +2,49 @@
 from django.contrib.auth.decorators import login_required
 from django.http import Http404
 from django.shortcuts import render
+from django.utils import timezone
 
 from ..auth.decorators import deny_restricted
+from .models import Route
 
 
 @login_required
 @deny_restricted
 def home(request):
+    now = timezone.localtime()
+    if now.hour > settings.BUS_PAGE_CHANGEOVER_HOUR:  # afternoon
```

Comment:
```suggestion
    if now.hour >= settings.BUS_PAGE_CHANGEOVER_HOUR:  # afternoon
```

This matches behavior of other pages.